docs/development/getting_started.rst

```diff
@@ -54,24 +54,34 @@ Either way, once you've installed Vagrant you should run:
 
 Finally, install Ansible so it can be used with Vagrant to automatically
 provision VMs. We recommend installing Ansible from PyPi with ``pip`` to ensure
-you have the latest stable version.
+you have the latest stable version:
 
 .. code:: sh
 
     sudo apt-get install python-pip
-    sudo pip install -U ansible
 
-If you're using Ubuntu, you can install a sufficiently recent version of
-Ansible from backports (if you prefer): ``sudo apt-get install
-ansible/trusty-backports``
+Since the version of ansible required to provision SecureDrop VMs may flux out
+of sync with the version in your distro's repos, we recommend installing the
+latest stable ansible to a virtual environment using `virtualenvwrapper` or
+`virtualenv`.
 
-.. note:: Tested: Ansible 1.9.4
+Using virtualenvwrapper:
 
-.. warning:: We do not recommend installing vagrant-cachier. It destroys apt's
-	     state unless the VMs are always shut down/rebooted with Vagrant,
-	     which conflicts with the tasks in the Ansible playbooks. The
-	     instructions in Vagrantfile that would enable vagrant-cachier are
-	     currently commented out.
+.. code:: sh
+
+    sudo apt-get install virtualenvwrapper
+    mkvirtualenv -p python2.7 securedrop
+    pip install ansible==1.8.4
+
+Using virtualenv (must be in SecureDrop base directory before running these
+commands):
+
+.. code:: sh
+
+    sudo apt-get install virtualenvwrapper
+    virtualenv -p python2.7 .
+    . bin/activate
+    pip install ansible==1.8.4
```

Comment:
Conor and I just talked about this in person, he did misread this section, but anyway, I do need to change the `apt-get` command in the second block to install virtualenv, not virtualenvwrapper.